cypress/support/step_definition/course/createNewCourse/createNewCourse.spec.js

```diff
@@ -1,37 +1,36 @@
 
-import Commom_Logins from '../../../page/common/logins'
-import Create_Course from '../../../page/course/createCoursePageObject'
-import Room_Overview from '../../../page/common/goToRoomOverview'
+import Commom_Logins from '../../../page/common/pageLogins'
+import Create_Course from '../../../page/course/pageCreateCoursePageObject'
+import Room_Overview from '../../../page/common/pageGoToRoomOverview'
+
+const commonLogin = new Commom_Logins()
+const roomOverview = new Room_Overview()
+const createCourse = new Create_Course()
+
 
 Given('I am logged in as a teacher', () => {
```

Comment:
This step should be moved to a file like step_definition\common\commonSteps.spec.js and then we don't need to add it everytime.
Did this already in my PR, so maybe we can ignore it for your PR.